Plugins/Cuda/include/Acts/Plugins/Cuda/Seeding2/SeedFinder.ipp

```diff
@@ -30,10 +32,12 @@ template <typename external_spacepoint_t>
 SeedFinder<external_spacepoint_t>::SeedFinder(
     Acts::SeedfinderConfig<external_spacepoint_t> commonConfig,
     const SeedFilterConfig& seedFilterConfig,
-    const TripletFilterConfig& tripletFilterConfig)
+    const TripletFilterConfig& tripletFilterConfig, int device,
+    Acts::Logging::Level loggerLevel)
```

Comment:
Can you consider accepting a logger instance here, and storing as a member variable? You can then default it to `Acts::getDefaultLogger`. This way, other logging backends (like then Athena logging for example) can potentially be passed in.

Reply:
Sure thing. I'll do that later today.

Reply:
If you look around, the pattern we usually use is accept an `std::unique_ptr<Logger>`, store as a member, and then provide a `const Logger& logger()` method that the macros call. But I'm sure you figured that out anyway 😉

Reply:
Done. Please have a look. :smile: